### DISCUSSION TITLE
Please add the ability to resize the card

### DISCUSSION THREAD

**Discussion by Leventionz** [slipx06/sunsynk-power-flow-card#221] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I cannot resize the card and would like to do this to better fit my dashboard

### Expected behaviour

The ability to specify height and width in px or something similar to this.

e.g.

type: custom:sunsynk-power-flow-card
cardstyle: lite
height: 300px
width: 300px

### Possible Solutions

None known at the moment other than implementing new feature

### Mode

Both

### Context / Reason

It is helpful to be able to resize the power flow card to meet user interface needs for different dashboard layouts

**Comment by slipx06** [comment]
Hi. You can scale the card using card_height and card_width attributes. They are not present in the visual editor. The [documentation](https://slipx06.github.io/sunsynk-power-flow-card/configuration.html) explains on how these can be used in conjunction with panel_mode. In order to preserve aspect ratio you cannot specify both width and height together.
